biome.json

```diff
@@ -1,8 +1,5 @@
 {
   "$schema": "node_modules/@biomejs/biome/configuration_schema.json",
-  "files": {
-    "ignoreUnknown": true
-  },
```

Comment:
This was needed to work around a bug in Biome, but that bug has now been fixed and is no longer needed!